src/services/firebaseServices.js

```diff
@@ -0,0 +1,66 @@
+import { db } from "../firebase/config";
+import { deleteDoc, doc, updateDoc } from "firebase/firestore";
+
+
+
+const updatePin=async(note)=>{
+    try {
+        await updateDoc(doc(db,"notes",note.id),{
+            isPinned:!note.isPinned,
+            // isArchive:!note.isArchive
+        })
+    } catch (error) {
+        console.log("error from updatePin",error)
```

Comment:
suggestion : instead of console log for error you can you console error

Reply:
sure thanks